### PR TITLE
Fix local dev env with datadog

### PR DESCRIPTION
**Pull request by jt-dd**
Fix local dev by adding the correct docker label.

### CODE REVIEW

**Review by edznux-dd — APPROVED** [review]
Does it fail silently if the `kubehound.env` is missing?

LGTM if so

File: deployments/kubehound/docker-compose.dev.ingestor.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ services:
       - "127.0.0.1:9000:9000"
     networks:
       - kubenet
+    env_file:
+      - kubehound.env
+    labels:
+      com.datadoghq.ad.logs: '[{"app": "grpc", "service": "kubehound"}]'
```
Comment (edznux-dd) [inline deployments/kubehound/docker-compose.dev.ingestor.yaml:15]: seems weird to have `app:grpc` tbh

**Comment by jt-dd** [comment]
> Does it fail silently if the `kubehound.env` is missing?
> 
> LGTM if so

Only the `kubehound.env.tpl` is pushed as it might contained secrets.

File: deployments/kubehound/docker-compose.dev.ingestor.yaml (hunk 1 above)
Comment (jt-dd) [reply deployments/kubehound/docker-compose.dev.ingestor.yaml:15]: I just put the name of container for the `app`. Same as all the others labels.
This is only for local dev anyway.

**Pull request merged**